starter-code/scripts/articleView.js

```diff
@@ -43,6 +46,7 @@ articleView.handleCategoryFilter = function() {
     $('#author-filter').val('');
   });
 };
+})
```

Comment:
Travis is failing due to a linter error (I think you have an unnecessary brace or parentheses here).